src/biogeochem/CNPhenologyMod.F90

```diff
@@ -1436,6 +1436,10 @@ subroutine CropPhenology(num_pcropp, filter_pcropp                     , &
     use pftconMod        , only : ntrp_corn, nsugarcane, ntrp_soybean, ncotton, nrice
     use pftconMod        , only : nirrig_trp_corn, nirrig_sugarcane, nirrig_trp_soybean
     use pftconMod        , only : nirrig_cotton, nirrig_rice
+    
+    ! Y. Cheng
+    use pftconMod        , only : nmiscanthus, nirrig_miscanthus, nswitchgrass, nirrig_switchgrass
```

Comment:
I'm assuming the new parameters are included in a parameter file that's not provided with this pull request, but are the parameter values sensible (for as much as we know about C4 grasses in CLM)?

Reply:
Yes, the parameter file I modified is clm5_params.c190829.nc. Can I also add the parameter file in the pull request?

> It's exciting to see this work moving forward.
> 
> I only quickly looked at the code and am not an expert on either the crop model or the land use datasets that could drive the model. Comments here are intended to start a conversation to review the science and code from the pull request.
> 
> @danicalombardozzi, do you have suggestions for naming the grainc_to_food flux to avoid confusion with the rest of crop harvest.
> 
> @lawrencepj1 My understanding is that this would not change historical simulations, as we don't have biofuels on the surface dataset. Are there future scenarios that call for biofuel expansions in SSP/RCP scenarios?
> 
> @yanyanchengHydro your single point simulations look promising. Do you have any plans to consider global-scale simulations to get a first look at the potential climate implications of biofuels (and to test the model can run globally)? Alternatively, what happens in future scenarios at the point scale?

@wwieder Thank you Will! We have run two future simulations with biofuel expansions (SSP2-RCP4.5 and SSP5-RCP8.5) at 0.125 degree resolution over the CONUS from 2005 to 2100. I plan to present it at the LMWG meeting on March.

Reply:
@yanyanchengHydro. Yes, you can include the paramfile in the Pull Request. It can be pointed to in the bld/namelist_files/namelist_defaults_ctsm.xml file. But, there will likely be permission issues. So do you have access to cheyenne? And could you point us to the file there?

Reply:
@wwieder we are working on releasing the files through a public facing serve so that we don't need to worry about transfer the parameter files back and forth. In addition to the clm5_params file, we will release the single point surfacedata, landuse, and forcing files so that the experiments are reproducible.

Reply:
@wwieder @ekluzek  We are working on releasing the files through a public facing server at PNNL so that we don't need to worry about transferring the files. In addition to the clm5_params file, we will release the single point surfacedata, landuse, and forcing files so that the experiments are reproducible. However, if it is easier, @yanyanchengHydro can put a copy on cheyenne.

Reply:
Just to clarify what @ekluzek said in case it wasn't clear: you can change the xml file but please do *not* add the netcdf file to your branch in git.